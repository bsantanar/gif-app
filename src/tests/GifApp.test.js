import { shallow } from 'enzyme'
import { GifApp } from '../GifApp'

describe('tests on <GifApp />', () => {
    

    test('should match snapshot', () => {
        const wrapper = shallow(<GifApp />)
        expect(wrapper).toMatchSnapshot()
    })
    
    test('should list category', () => {
        const categories = ['saitama', 'tanjiro']
        const wrapper = shallow(<GifApp defaultCategories={categories} />)

        expect(wrapper.find('GifGrid').length).toBe(categories.length)

    })
    

})
