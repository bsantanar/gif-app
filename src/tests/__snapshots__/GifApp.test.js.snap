// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests on <GifApp /> should match snapshot 1`] = `
<div>
  <h2>
    Gif App
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol />
</div>
`;
